server/lib/slack.js

```diff
@@ -9,9 +9,11 @@ const createPayload = (report, extensionUrl) => {
     attachments: []
   };
 
+  const providerName = process.env.A0EXT_PROVIDER.charAt(0).toUpperCase() + process.env.A0EXT_PROVIDER.slice(1);
```

Comment:
could `process.env.A0EXT_PROVIDER` be `undefined`? if so, `process.env.A0EXT_PROVIDER.charAt(0)` will throw an error.

Reply:
actually, we _already_ have `lodash` pulled into this file. might as well just use `_.capitalize(process.env.A0EXT_PROVIDER)` and save ourselves the hassle 😉 

https://lodash.com/docs/4.17.11#capitalize